editors/vscode/tests/e2e_server.spec.ts

```diff
@@ -341,4 +151,73 @@ suite('E2E Diagnostics', () => {
     strictEqual(nestedDiagnostics[0].code.target.authority, 'oxc.rs');
     strictEqual(nestedDiagnostics[0].severity, DiagnosticSeverity.Error);
   });
+
+  testMultiFolderMode('different diagnostic severity', async () => {
+    await loadFixture('debugger');
+    await loadFixture('debugger_error', WORKSPACE_SECOND_DIR);
+
+    const firstDiagnostics = await getDiagnostics('debugger.js');
+    const secondDiagnostics = await getDiagnostics('debugger.js', WORKSPACE_SECOND_DIR);
+
+    assert(typeof firstDiagnostics[0].code == 'object');
+    strictEqual(firstDiagnostics[0].code.target.authority, 'oxc.rs');
+    strictEqual(firstDiagnostics[0].severity, DiagnosticSeverity.Warning);
+
+    assert(typeof secondDiagnostics[0].code == 'object');
+    strictEqual(secondDiagnostics[0].code.target.authority, 'oxc.rs');
+    strictEqual(secondDiagnostics[0].severity, DiagnosticSeverity.Error);
+  })
+
+  test('cross module', async () => {
```

Comment:
I needed to move the test to the end to work. I do not have an idea why, and it is already frustrating enough to create tests^^